src/Components/Form/index.jsx

```diff
@@ -2,20 +2,20 @@ import { useState } from 'react';
 import './Form.scss';
 
 function Form (props) {
-    const [url, setUrl] = useState('');
+    const [url, setUrl] = useState('https://pokeapi.co/api/v2/pokemon');
```

Comment:
We probably don't want start with anything populated here, unless we make it a controlled form so that the value visibly matches the state.